src/test/groovy/com/morpheusdata/scvmm/MainPackageLogWrapperSpec.groovy

```diff
@@ -0,0 +1,82 @@
+// (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+
+// Create a test class in the main scvmm package (not logging sub-package) 
+// to ensure package prefix matching works correctly
```

Comment:
Can you check if this removing this class reduces any code coverage. If not please remove this testing class.